src/template.js

```diff
@@ -35,12 +44,37 @@ switch (data.method) {
       break;
     }
 
-    if (queryPermission('inject_script', INSIGHTS_LIBRARY_URL)) {
+    const url = getLibraryURL(data.useIIFE);
+
+    if (queryPermission('inject_script', url)) {
       injectScript(
-        INSIGHTS_LIBRARY_URL,
-        data.gtmOnSuccess,
+        url,
+        () => {
+          if (!copyFromWindow('aa')) {
```

Comment:
If even `window.aa` doesn't exist, then it's definitely wrong. But I'm not sure if this is really possible. Leaving this error log just in case.